{
  "id": "EUVD-2025-31130",
  "enisaUuid": "33db06ce-4c8f-3f0f-93d5-cfbdd716e82b",
  "description": "pytorch v2.8.0 was discovered to contain an integer overflow in the component torch.nan_to_num-.long().",
  "datePublished": "Sep 25, 2025, 6:30:34 PM",
  "dateUpdated": "Sep 26, 2025, 9:30:27 PM",
  "baseScore": 5.3,
  "baseScoreVersion": "3.1",
  "baseScoreVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:L",
  "references": [
    "https://github.com/pytorch/pytorch/issues/151510",
    "https://gist.github.com/shaoyuyoung/0e7d2a586297ae9c8ed14d8706749efc",
    "https://nvd.nist.gov/vuln/detail/CVE-2025-55554"
  ],
  "aliases": [
    "CVE-2025-55554"
  ],
  "assigner": "mitre",
  "epss": 0.05,
  "enisaIdProduct": [
    {
      "id": "b6f07783-de43-39e6-9556-46908847693d",
      "product": {
        "name": "n/a"
      },
      "product_version": "n/a"
    }
  ],
  "enisaIdVendor": [
    {
      "id": "48e15a53-744b-3be3-86bf-bacc94b7fbbc",
      "vendor": {
        "name": "n/a"
      }
    }
  ]
}